accountsdb-plugin-postgres/src/accountsdb_plugin_postgres.rs

```diff
@@ -1,27 +1,30 @@
 /// Main entry for the PostgreSQL plugin
 use {
-    crate::accounts_selector::AccountsSelector,
+    crate::{
+        accounts_selector::AccountsSelector,
+        postgres_client::{
+            ParallelPostgresClient, PostgresClient, PostgresClientBuilder, SimplePostgresClient,
+        },
+    },
     bs58,
-    chrono::Utc,
     log::*,
-    postgres::{Client, NoTls, Statement},
     serde_derive::{Deserialize, Serialize},
     serde_json,
     solana_accountsdb_plugin_interface::accountsdb_plugin_interface::{
         AccountsDbPlugin, AccountsDbPluginError, ReplicaAccountInfoVersions, Result, SlotStatus,
     },
-    std::{fs::File, io::Read, sync::Mutex},
+    std::{fs::File, io::Read},
     thiserror::Error,
 };
 
-struct PostgresSqlClientWrapper {
-    client: Client,
-    update_account_stmt: Statement,
+enum PostgresClientEnum {
+    Simple(SimplePostgresClient),
```

Comment:
Can this `SimplePostgresClient` this just be replaced with a `ParallelPostgresClient` of only 1 worker thread?

Reply:
There is a subtle but important difference. Using the SimplePostgresClient -- the account info input is not cloned because it does not need to be sent over to different threads. It would be more performant that the single threaded ParallelPostgresClient which still requires cloning.